src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/lettertoprison/LetterToPrisonCustodyOfficeGenerator.kt

```diff
@@ -38,10 +38,10 @@ class LetterToPrisonCustodyOfficeGenerator(
         setVariable("recallLengthDays", RecallLengthDescription(this.recallLength).numberOfDays())
         setVariable("hasAdditionalLicenceConditions", this.additionalLicenceConditions)
         setVariable("additionalLicenceConditionsDetail", this.additionalLicenceConditionsDetail)
-        setVariable("hasVulnerabilities", this.vulnerabilityDiversityDetail?.isNotEmpty())
-        setVariable("vulnerabilityDiversityDetail", this.vulnerabilityDiversityDetail)
-        setVariable("hasContraband", this.contrabandDetail?.isNotEmpty())
+        setVariable("hasContraband", this.contraband ?: false)
```

Comment:
Should we be defaulting this? Feels like we should force it (!!) as its an expected value.

Reply:
Done.